packages/twenty-front/src/modules/activities/hooks/useCustomResolver.ts

```diff
@@ -0,0 +1,102 @@
+import { useState } from 'react';
+import { useQuery } from '@apollo/client';
+
+import { ActivityTargetableObject } from '@/activities/types/ActivityTargetableEntity';
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+
+export const useCustomResolver = (
+  query: any,
```

Comment:
Don't we have a generic interface for queries ? same for data ?

Reply:
Yes, I wanted to do it later but I forgot